src/main/java/com/apptware/interview/jpa/employee/Employee.java

```diff
@@ -2,14 +2,33 @@
 
 import jakarta.persistence.Entity;
 import java.util.UUID;
+
+import jakarta.persistence.Id;
 import lombok.Getter;
 import lombok.Setter;
 
-@Getter
-@Setter
+//@Getter
+//@Setter
 @Entity
 class Employee {
 
+  @Id
```

Comment:
test case of EmplyeeTest were not working because here @Id was missing. so added that.